mindmeld/tokenizer.py

```diff
@@ -171,9 +175,21 @@ def _init_regex(self):
         }
 
         # Create a regular expression  from the dictionary keys
-        self.keep_special_compiled = re.compile(
-            "(%s)" % ")|(".join(keep_special_regex_list), re.UNICODE
-        )
+        # and add app-specific regex config to the list if specified
+
+        if self.config["patterns"]:
+            self.keep_special_compiled = re.compile(
+                "((%s))&(%s)"
+                % (
+                    ")|(".join(keep_special_regex_list),
+                    ")|(".join(self.config["patterns"]),
```

Comment:
So here the whitelist set of chars are combined with the `keep_special_regex_list `? Whats the regex pattern here? Is it, if the `keep_special_regex_list ` set of chars or `self.config["patterns"]` patterns match, whitelist, ie allow them?

Reply:
Yes. In the event of default configuration we continue with the current mindmeld regex, otherwise a combination of the current regex and the new pattern. 

Alternatively we can also do the following: in the case of a developer-defined tokenizer config, only apply that and not the default MM special chars list.